app-editors/ghostwriter/ghostwriter-1.7.4-r1.ebuild

```diff
@@ -7,7 +7,7 @@ inherit qmake-utils xdg-utils
 
 DESCRIPTION="Cross-platform, aesthetic, distraction-free markdown editor"
 HOMEPAGE="https://wereturtle.github.io/ghostwriter/"
-SRC_URI="https://github.com/wereturtle/ghostwriter/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/wereturtle/ghostwriter/archive/v${PV}.tar.gz -> ${PF}.tar.gz"
```

Comment:
no need to change anything here, in fact `PF` is wrong as a revbump does not change anything on our tarball, does it?